test/cql-pytest/run.py

```diff
@@ -266,7 +266,15 @@ def get_cql_cluster(ip, ssl_context=None):
     auth_provider = cassandra.auth.PlainTextAuthProvider(username='cassandra', password='cassandra')
     return cassandra.cluster.Cluster([ip],
         auth_provider=auth_provider,
-        ssl_context=ssl_context)
+        ssl_context=ssl_context,
+        # The default timeout for new connections is 5 seconds, and for
+        # requests made by the control connection is 2 seconds. These should
+        # have been more than enough, but in some extreme cases with a very
+        # slow debug build running on a very busy machine, they may not be.
+        # so let's increase them to 60 seconds. See issue #11289.
+        connect_timeout = 60,
+        control_connection_timeout = 60,
+        max_schema_agreement_wait=60)
```

Comment:
I think there is no need to change this file (run.py). This code checks in a loop if Scylla is already up, so if one of the connections times out, it will simply try again.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/c1ebd7e95589bb313679f526c10045f6a1dcbe81